### ISSUE TITLE
not_valid_after property is invoked as a method producing errors

### ISSUE THREAD

**Issue by kyahehron** [m-click/requests_pkcs12#29]
@vog  I believe `not_valid_after` is a property that returns `datetime.datetime` object, not a callable?

```
line 37, in check_cert_not_after
   cert_not_after = cert.not_valid_after()
TypeError: 'datetime.datetime' object is not callable
```
Commit is from [cb09e98](https://github.com/m-click/requests_pkcs12/commit/cb09e98bc4a6aa4d7c776822a6f59c9ed422f735)

REF: https://github.com/pyca/cryptography/blob/87f43fb77e7f1c56471d3233a56000a51dc4a311/src/cryptography/x509/base.py#L118

**Comment by vog** [comment]
Thanks! I just fixed that and released it as version 1.12 on PyPI.

Does this solve it for you?

**Comment by kyahehron** [comment]
Unfortunately no, the commit brought another issue found here #30

**Comment by vog** [comment]
Closing this, as the problem at hand is solved. The other issue is discussed (and meanwhile solved) in #30.
